### PR TITLE
Appeal listener

### PR DESCRIPTION
**Pull request by TymanWasTaken**
Automattically fetches user and modlogs for submitted appeals and creates a thread with it

### CODE REVIEW

File: src/listeners/bush/appealListener.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		const user = await this.client.users.fetch(userId).catch(() => null);
+		if (!user) {
+			await thread.send({
+				embeds: [
+					this.client.util
+						.embed()
+						.setTitle(
+							`${message.embeds[0].fields!.find((f) => f.name === 'What type of punishment are you appealing?')!.value} appeal`
+						)
+						.addField({
```
Comment (IRONM00N) [inline src/listeners/bush/appealListener.ts:29]:
```suggestion
						.addFields({
```
discord.js removed addField

File: src/listeners/bush/appealListener.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+				where: {
+					user: user.id,
+					guild: message.guildId
+				},
+				order: [['createdAt', 'DESC']]
+			});
+			await thread.send({
+				embeds: [
+					this.client.util
+						.embed()
```
Comment (IRONM00N) [inline src/listeners/bush/appealListener.ts:46]: no gross

**Review by IRONM00N — CHANGES_REQUESTED** [review]

File: src/listeners/bush/appealListener.ts (hunk 2 above)
Comment (TymanWasTaken) [reply src/listeners/bush/appealListener.ts:46]: ??????

Comment (IRONM00N) [reply src/listeners/bush/appealListener.ts:46]: just import Embed

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**